cleansio/speech/transcribe.py

```diff
@@ -37,4 +37,5 @@ def print_transcription(response, index, length):
     if not response.results:
         return
     result = response.results[0].alternatives[0].transcript
-    print(leading_zero(index) + '/' + leading_zero(length) + ': ' + result)
+    #print(leading_zero(index) + '/' + leading_zero(length) + ': ' + result)
+    print(result)
```

Comment:
Debugging?

Reply:
Yes - the leading zeros were throwing off my similarity measurement, as the Python script I used counted them as differing "words".